### ISSUE TITLE
Improve melanoma classification accuracy

### ISSUE THREAD

**Issue by alif-munim** [alif-munim/r-tensorflow#12]
In the first pass attempt at melanoma classification, the 2-layer neural network trained on ~150 images only achieved a ~30% accuracy. Try again with more training data and perhaps a deeper network.
